src/libraries/System.Private.CoreLib/src/System/Reflection/AssemblyName.cs

```diff
@@ -83,14 +83,14 @@ public ProcessorArchitecture ProcessorArchitecture
             get
             {
                 int x = (((int)_flags) & 0x70) >> 4;
-                if (x > 5)
+                if (x > 6)
```

Comment:
The changes in this file should be reverted.

Reply:
OK,Thanks.